calendar_resource/models/calendar_event.py

```diff
@@ -13,3 +15,31 @@ class CalendarEvent(models.Model):
         string='Resources',
         comodel_name='resource.resource',
     )
+
+    @api.constrains('resource_ids')
+    def _check_resource_ids_dbl_book(self):
+        """ Check Double Booking """
+        for record in self:
+
+            resources = record.resource_ids.filtered(
+                lambda s: s.allow_double_book is False
+            )
+
+            if not any(resources):
+                continue
+
+            overlaps = self.env['calendar.event'].search([
+                ('id', '!=', record.id),
+                ('start', '<', record.stop),
+                ('stop', '>', record.start),
```

Comment:
Hmmm. Well if you had a resource in an appointment ending at 5pm and another starting at 5pm, would you consider that double booking the resource?